CONTRIBUTING.md

```diff
@@ -1,5 +1,9 @@
 # Contributing
 
+## Requirements
+
+- `brew install jq` for OSX (for other systems please look up [here](https://stedolan.github.io/jq/)) is required to run `lerna audit run`
```

Comment:
```suggestion
- `brew install jq` for OSX (for other systems please look up [here](https://stedolan.github.io/jq/)) is required to run `npm run audit` or `lerna audit run`.
```

Looks good to me. Do we want to pull up the section about the postgres packages (first paragraph in "Local Set Up") up here, feels like both should be listed under "Requirements" if we have this section. WDYT?

Reply:
Yeah was wondering about this as well. Let's combine the sections and simply use "Requirements".